### PR TITLE
Hosting Tab

### PR DESCRIPTION
**Pull request by Mingling94**
Hosting tab now shows total and remaining space.
sia_002

### CODE REVIEW

**Comment by DavidVorick** [comment]
We need to confirm that it properly talks to the daemon when updating these values.

**Comment by Mingling94** [comment]
That was something I didn't understand exactly what was going on, but the
new figures operate in the same manner as the old ones, so it should work.

On 12:19PM, Wed, Jun 3, 2015 David Vorick <notifications@github.com> wrote:

> We need to confirm that it properly talks to the daemon when updating
> these values.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/NebulousLabs/Sia-UI/pull/124#issuecomment-108508517>.
>
-- 
Sincerely and cheerily,

Ming Luo
Rensselaer Polytechnic Institute '15
B.S. Computer Science
Cell: (434) 420-2952
mcluo1994@gmail.com

**Comment by Mingling94** [comment]
Whoops, didn't mean to delete that branch

File: site/markup/pages/files.haml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #files
     .header
-        .title File Manager
+        .title Files Manager
```
Comment (DavidVorick) [inline site/markup/pages/files.haml]: Should it be plural?

File: site/markup/pages/hosting.haml [hunk 2]
```diff
@@ -1,9 +1,10 @@
 #hosting
     .header
-        .title Host Management
+        .title Hosting Management
```
Comment (DavidVorick) [inline site/markup/pages/hosting.haml]: 'Host Management' is not better?

File: site/scripts/util.js [hunk 3]
```diff
@@ -1,28 +1,27 @@
...
-        var degree = Math.floor(Math.log(Math.abs(number)) / Math.LN10 / 3);
-
-        var numberString = String(number / Math.pow(1000,degree));
-
-        var si = degree > 0 ? pSI[degree] : nSI[degree * -1];
-
-        return numberString.slice(0,precision + 1) + " " + si;
+    // Siacoin to USD
+    function USDConvert(balance){
+        return balance * 0.0000000172;
```
Comment (DavidVorick) [inline site/scripts/util.js]: This should be removed

File: site/markup/pages/files.haml (hunk 1 above)
Comment (lukechampine) [inline site/markup/pages/files.haml]: yeah I thought the same.

File: site/scripts/util.js (hunk 3 above)
Comment (Mingling94) [inline site/scripts/util.js]: USDConvert? or engineering notation?

Comment (DavidVorick) [inline site/scripts/util.js]: USDConvert

File: site/scripts/util.js [hunk 4]
```diff
@@ -34,21 +33,52 @@ var util = (function(){
...
+    }
+
+    // precision determines the length of number
+    function round(number, flex){
+        if (!flex) flex = .01;
+        if (Math.ceil(number) - number <= flex)
+            return Math.ceil(number);
+        else if (number - Math.floor(number) <= flex)
+            return Math.floor(number);
+    }
```
Comment (DavidVorick) [inline site/scripts/util.js]: can't you just use number.toFixed(n)?

Does this do something different?

**Comment by DavidVorick** [comment]
This is good for now, the remaining changes are much closer to 'whole codebase needs refactoring' type stuff than 'is a fundamental part of this PR' type stuff.

**Pull request merged**